src/api/discogsApi.js

```diff
@@ -1,17 +1,20 @@
-const getArtists = async () => {
-  const response = await fetch(`https://api.discogs.com/artists/1?key=QaxrqhMkbuCukhWNmnUq&secret=nWRmusdJoUlFqzzXLpbKhMtsGOlGEGGe`)
-  if (!response.ok) {
-    if (response.status >= 500) {
-      throw new Error(`${response.status}. I am so sorry for the inconvenience! The server is down. Please try again, later.`)
-    } else if (response.status === 404) {
-      throw new Error(`${response.status}. We apologize for the inconvenience. Please try again, or go back to the Main Page.`)
-    } else {
-      throw new Error(`${response.status}. We apologize for the inconvenience. Please try again later.`)
-    }
-  } else {
-    return response.json()
-  }
-    
+const getArtists = async (artistsId) => {
+
+  return fetch(`https://api.discogs.com/artists/${artistsId}?key=QaxrqhMkbuCukhWNmnUq&secret=nWRmusdJoUlFqzzXLpbKhMtsGOlGEGGe`)
```

Comment:
You might get a message from GitHub about having the api key in your code that youre pushing up. Sometimes these need to be private.